service-tests/eclipse-marketplace.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+const {
+	  isMetric,
+	  isMetricOverPeriod,
+	  isVPlusDottedVersionAtLeastOne
+} = require('./helpers/validators');
+
+const t = new ServiceTester({ id: 'eclipse-marketplace', title: 'Eclipse' });
+module.exports = t;
+
+t.create('total marketplace downloads')
+  .get('/d/notepad4e.json')
```

Comment:
Could you make this `/dt/` for consistency with the other badges?

Reply:
I thought I was being consistent here, but after a closer look /d/ is used alone only when there are no yearly/monthly/weekly statistics for the same badge. Will fix that.